### ISSUE TITLE
Google Chrome Lighthouse Scan: Issues Detected

### ISSUE THREAD

**Issue by nikolayg** [mmistakes/so-simple-theme#434]
## Environment

The demo pages linked from the main repo - e.g. [https://mmistakes.github.io/so-simple-theme/](https://mmistakes.github.io/so-simple-theme/)

## Expected behavior

I ran the Google Chrome Lighthouse scan on the sample page. There were a few reported issues with respect to accessibility, performance, and security. For example the `Previous` button does not have enough contrast with the background and an old version of JQuery is used (which has vulnerabilities):

![Screen Shot 2021-10-05 at 10 39 07 pm](https://user-images.githubusercontent.com/4774199/136016189-1d1377b6-236d-4401-8be2-fdebbfdee374.png)

## Steps to reproduce the behavior

Open the sample page https://mmistakes.github.io/so-simple-theme/ and run the Chrome Lighthouse tool.
